### ISSUE TITLE
lighty: MaintTest.startConfigFileTest FAILURE

### ISSUE THREAD

**Issue by purplezimmermann** [purplezimmermann/opendaylight-transportpce#3]
```java
[ERROR] startConfigFileTest on startConfigFileTest(io.lighty.controllers.tpce.MaintTest)(io.lighty.controllers.tpce.MaintTest)  Time elapsed: 21.087 s  <<< FAILURE!
java.lang.NoSuchMethodError: 'java.lang.Object org.opendaylight.yangtools.yang.binding.CodeHelpers.requireKeyProp(java.lang.Object, java.lang.String)'
        at io.lighty.controllers.tpce.MaintTest.startConfigFileTest(MaintTest.java:60)
```

`CodeHelpers.requireKeyProp` was added in `yang-binding` ~~**7.0.6**~~ via https://github.com/opendaylight/mdsal/commit/1ee8e403b9c9eb9e8596b5e89cc5ed47676eb71c

~~_**But!**_~~ All current `mdsal` dependencies are set to **7.0.5**

**Comment by purplezimmermann** [comment]
~~Resolved by #4~~

**Comment by purplezimmermann** [comment]
Just have tested before claiming that this is solved by #4! _**It is not!**_

`CodeHelpers.requireKeyProp` is available in all `mdsal:yang-binding` **7.0.x**

There must be an issue with the `mdsal:binding-parent` and `mdsal-artifacts` **7.0.x** dependencies, because running `mvn  dependency:tree` reveals several `org.opendaylight.*:jar:6.0.4:compile - version managed from 7.0.6`

**Comment by purplezimmermann** [comment]
Resolved by #5
